slate/source/index.html.md

```diff
@@ -424,7 +423,7 @@ This endpoint must be hosted on a website with TLS protection (HTTPS).
 
 ## Client-Initiated Backchannel Authentication (CIBA)
 Client Initiated Backchannel Authentication (CIBA) enables a Data Recipient to
-initiate the authentication of an end-user at a Data Holder by means of out-band
```

Comment:
out-band is the correct term see https://openid.net/specs/openid-connect-modrna-client-initiated-backchannel-authentication-1_0.html